pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/etterlatte/maler/vedlegg/barnepensjon/BeregningAvBarnepensjon.kt

```diff
@@ -196,10 +215,18 @@ private fun OutlineOnlyScope<LanguageSupport.Triple<Bokmal, Nynorsk, English>, B
         )
     }
     paragraph {
-        textExpr(
-            Bokmal to "Barnepensjonen utgjør en ganger folketrygdens grunnbeløp (G). Folketrygdens grunnbeløp er per i dag ".expr()  + grunnbeloep.format() + " kroner. Grunnbeløpet blir regulert 1. mai hvert år. Økningen etterbetales vanligvis i juni hvert år. ",
-            Nynorsk to "Barnepensjonen per år utgjer éin gong grunnbeløpet i folketrygda (G).  Grunnbeløpet i folketrygda er per i dag  ".expr()  + grunnbeloep.format() + " kroner.  Grunnbeløpet blir regulert 1. mai kvart år. Auken blir vanlegvis etterbetalt i juni kvart år.",
-            English to "The children's pension per year amounts to 1 x the national insurance basic amount (G).  The national insurance basic amount currently amounts to ".expr()  + grunnbeloep.format() + " kroner.  The basic amount is adjusted on 1 May each year. You will receive payment of any increase in June of each year. ",
+        ifElse(
+            erForldreloes,
+            textExpr(
+                Bokmal to "Når begge foreldrene er døde, blir den årlige pensjonen lik 2,25 ganger grunnbeløpet. Folketrygdens grunnbeløp per i dag er ".expr() + grunnbeloep.format() + " kroner. Dette deles på 12 måneder. Grunnbeløpet blir regulert 1. mai hvert år. Økningen etterbetales vanligvis i juni hvert år.",
+                Nynorsk to "".expr(),
+                English to "".expr()
+            ),
+            textExpr(
+                Bokmal to "Barnepensjonen utgjør en ganger folketrygdens grunnbeløp (G). Folketrygdens grunnbeløp er per i dag ".expr()  + grunnbeloep.format() + " kroner. Grunnbeløpet blir regulert 1. mai hvert år. Økningen etterbetales vanligvis i juni hvert år. ",
+                Nynorsk to "Barnepensjonen per år utgjer éin gong grunnbeløpet i folketrygda (G).  Grunnbeløpet i folketrygda er per i dag  ".expr()  + grunnbeloep.format() + " kroner.  Grunnbeløpet blir regulert 1. mai kvart år. Auken blir vanlegvis etterbetalt i juni kvart år.",
+                English to "The children's pension per year amounts to 1 x the national insurance basic amount (G).  The national insurance basic amount currently amounts to ".expr()  + grunnbeloep.format() + " kroner.  The basic amount is adjusted on 1 May each year. You will receive payment of any increase in June of each year. ",
```

Comment:
bruker vanligvis det multiplikasjonstegnet i stedet for vanlig x her.

Reply:
Dette er en linje jeg bare har flyttet inn i en if. Så har vært slik i prod til nå. Mener du at jeg fortsatt skal endre?